### PR TITLE
Storybook version Update

### PR DESCRIPTION
**Pull request by spark3-mdb**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

**Comment by spark3-mdb** [comment]
Going to wait until #1195 is merged, since this will also be updating yarn.lock.

**Review by shaneeza — APPROVED** [review]
As we spoke about, the knobs are a little finicky, I have to hit `reset` to see the correct knobs but if this is only temporary I think its ok.

**Pull request merged**
